src/app/api/stacks/clients/stx20-api-client.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+export function stx20ApiClient() {
+  return axios.create({
+    baseURL: 'https://api.stx20.com/api/v1',
+  });
+}
```

Comment:
I think we want to call this once, and export, vs every time. 

E.g, if we want to modify all requests for stx20.com, we'd use `stx20Client.interceptors`, which afaik depends on a single instance

Reply:
mm, but yeah we can't do this for custom networks

Reply:
so not I'm questioning if it is a good idea to use all the separate axios clients / vs a single client where we don't set baseURL. Thoughts?

Reply:
Thinking on all the refactor here, I might close this and separate out the STX-20 balance work just to get that merged first. I'm not sure we absolutely need to refactor the api clients rn, but I'd like to think on how to set this all up a bit better.

Reply:
You are right, this should only be called once with the config nec to handle that service.
https://blog.logrocket.com/understanding-axios-create/